### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by joseph-w-valdez**

### CODE REVIEW

**Comment by joseph-w-valdez** [comment]
https://user-images.githubusercontent.com/117682160/209734053-41817038-9b24-48f1-b216-6b681081339c.mp4

**Comment by joseph-w-valdez** [comment]
image

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+$submit.addEventListener('submit', handleSubmit);
+
+var $title = document.querySelector('#title');
+var $photoUrl = document.querySelector('#photo');
+var $notes = document.querySelector('#notes');
+
+var $previewImage = document.querySelector('img');
+function handlePreview(event) {
+  event.preventDefault();
```
Comment (codingcodymiller) [inline js/main.js:26]: `preventDefault` isn't necessary for the `input` event

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  entry.title = $title.value;
+  entry.photoUrl = $photoUrl.value;
+  entry.notes = $notes.value;
+  entry.entryId = data.nextEntryId;
+  data.nextEntryId = data.nextEntryId + 1;
+  data.entries.unshift(entry);
+  $previewImage.setAttribute('src', './images/placeholder-image-square.jpg');
+  $title.value = null;
+  $photoUrl.value = null;
+  $notes.value = null;
```
Comment (codingcodymiller) [inline js/main.js:15]: You can use the `reset` method on the `form` element

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+
+var $submit = document.querySelector('form');
```
Comment (codingcodymiller) [inline js/main.js:2]: You should rename this variable to `$form` so you don't mistakenly think it is the submit button

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**